### PR TITLE
[Portal] Add chromatic story

### PR DESCRIPTION
**Pull request by benoitgrelard**

### CODE REVIEW

File: packages/react/portal/src/Portal.stories.tsx [hunk 1]
```diff
@@ -53,3 +54,74 @@ export const CustomContainer = () => {
...
+      </div>
+
+      <h1>Custom container</h1>
+      <div style={{ padding: 10, margin: 10, border: '1px solid green' }}>
+        <p>Container B</p>
+        <Portal containerRef={portalContainerRef}>
+          <div style={{ padding: 10, margin: 10, border: '1px solid green' }}>
+            <p>
+              This content is rendered in a portal inside Container B but appears inside Container B
+              because we have used Container C as a container element for the Portal.
```
Comment (jjenzz) [inline packages/react/portal/src/Portal.stories.tsx:86]: Should this say the following?

```diff
- This content is rendered in a portal inside Container B but appears inside Container B 
+ This content is rendered in a portal inside Container B but appears inside Container C 
```

That seems to be where it is appearing 🤔

**Review by jjenzz — APPROVED** [review]
Small copy change but otherwise :+1:

File: packages/react/portal/src/Portal.stories.tsx (hunk 1 above)
Comment (benoitgrelard) [reply packages/react/portal/src/Portal.stories.tsx:86]: Yep, my bad, I changed that copy from the copy/paste of the old stories but forgot to update that one letter 😀

File: packages/react/portal/src/Portal.stories.tsx [hunk 2]
```diff
@@ -53,3 +54,74 @@ export const CustomContainer = () => {
...
+        </Portal>
+      </div>
+
+      <h1>Custom container</h1>
+      <div style={{ padding: 10, margin: 10, border: '1px solid green' }}>
+        <p>Container B</p>
+        <Portal containerRef={portalContainerRef}>
+          <div style={{ padding: 10, margin: 10, border: '1px solid green' }}>
+            <p>
+              This content is rendered in a portal inside Container B but appears inside Container B
```
Comment (benoitgrelard) [inline packages/react/portal/src/Portal.stories.tsx:85]:
```suggestion
              This content is rendered in a portal inside Container B but appears inside Container C
```

**Pull request merged**